db/install.xml

```diff
@@ -40,7 +41,7 @@
         <FIELD NAME="course" TYPE="int" LENGTH="20" NOTNULL="true" UNSIGNED="true" SEQUENCE="false" PREVIOUS="id" NEXT="coursefeedbackid"/>
         <FIELD NAME="coursefeedbackid" TYPE="int" LENGTH="10" NOTNULL="true" UNSIGNED="true" SEQUENCE="false" PREVIOUS="course" NEXT="questionid"/>
         <FIELD NAME="questionid" TYPE="int" LENGTH="10" NOTNULL="true" UNSIGNED="true" SEQUENCE="false" PREVIOUS="coursefeedbackid" NEXT="answer"/>
-        <FIELD NAME="answer" TYPE="int" LENGTH="4" NOTNULL="true" UNSIGNED="true" SEQUENCE="false" PREVIOUS="questionid" NEXT="timemodified"/>
+        <FIELD NAME="answer" TYPE="int" LENGTH="4" NOTNULL="false" UNSIGNED="true" SEQUENCE="false" PREVIOUS="questionid" NEXT="textanswer"/>
```

Comment:
NEXT stimmt nicht. Da previous/next nur optionale Attribute sind, würde ich die einfach ganz weglassen.

Außerdem bildest du die Veränderung von NOTNULL in der upgrade.php nicht ab.

Reply:
Die Änderung ist fälschlicherweise von einer vorangegangenen Idee mit rein gerutscht wo ich dachte ich füge der Tabelle ein "textanswer" Feld hinzu anstelle eine eigene Tabelle zu machen.